HelpSource/Classes/UnitTest.schelp

```diff
@@ -78,7 +78,7 @@ within it, e.g. code::"TestPolyPlayerPool:test_prepareChildrenToBundle"::.
 
 
 METHOD:: runAll
-runs all subclasses of UnitTest
+Run the entire test suite. As this method addresses the entire test suite, it should be called only on code::UnitTest::, and not on its subclasses, i.e. individual unit tests.
```

Comment:
this last part doesn't make sense to me. a subclass of `UnitTest` isn't an individual unit test, it's a test class which may have 0 or more individual unit tests. also, probably good to note that trying to do that will result in an error.